README.md

```diff
@@ -73,6 +73,43 @@ end
   HTTP status 400 and above will be logged with `:error`, and redirect with `:warn`.
 * `:log_tag` - The log tag to be prefixed in the logs, default to: `#{inspect(__MODULE__)}`.
 
+## MetaLogger.Formatter protocol
+It is possible to define an implementation for a custom struct, so MetaLogger will know how to format log messages. It also includes the possibility to filter some data using regexp patterns.
+It could be useful, when there is defined a struct with sensitive information, for example after an HTTP request.
```

Comment:
If your intention was to add a paragraph, we have to add a blank line before. Otherwise the text remains in the same line. 

![image](https://user-images.githubusercontent.com/79254/120348229-48461900-c2fd-11eb-840d-17f3c0c603e4.png)